home/home3.2/Triangle.java

```diff
@@ -0,0 +1,53 @@
+class Constants {
+        public static final char a ='a';
+        public static final char b = 'b';
+        public static final char c = 'c';
+    }
+
+
+public class Triangle {
+    private double a;
+    private double b;
+    private double c;
+    public Triangle(double a, double b, double c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+    }
+
+    public String getArea() {
+        if (isValid()) {
+            return String.format("%.2f", calculateArea());
+        } else {
+            return "triangle is invalid";
+        }
+    }
+
+    public String getHeight(char side) {
```

Comment:
we can use enum for side detection and it allows us to validate user input and convert it to enum value before we even call this method